docs/quickstart.md

```diff
@@ -0,0 +1,195 @@
+
+=== "Linux"
+    Open up your command line interface and install the required dependencies:
+
+    ```
+    sudo apt-get install git build-essential cmake libmbedtls-dev zlib1g-dev
+    ```
+
+    Create a new directory, e.g. called `AVSystem`
+
+    ```
+    mkdir ~/AVSystem \
+        && cd AVSystem
+    ```
+
+    Clone the Anjay GitHub repository and compile the Anjay demo client using Cmake.
+
+    ```
+    git clone https://github.com/AVSystem/Anjay.git \
+        && cd Anjay \
+        && git submodule update --init \
+        && cmake . \
+        && make -j
+    ```
+
+=== "Mac"
+    Open up your terminal and install the required dependencies using [Homebrew](https://brew.sh/).
+
+    ```
+    brew install cmake mbedtls
+    ```
+
+    Create a new directory, e.g. called `AVSystem`
+
+    ```
+    mkdir ~/AVSystem \
+        && cd AVSystem
+    ```
+
+    Clone the Anjay GitHub repository and compile the Anjay demo client using Cmake.
+
+    ```
+    git clone https://github.com/AVSystem/Anjay.git \
+        && cd Anjay \
+        && git submodule update --init \
+        && cmake . \
+        && make -j
+    ```
+
+=== "Windows"
+    **Install the required dependencies**
+
+    1. Install [MSYS2](http://www.msys2.org/)
+    
+    1. Install [Git for Windows](https://gitforwindows.org/)
+    
+        !!! Note 
+            You can also install these using [Chocolatey](https://chocolatey.org/): `choco install git msys2` but make sure to follow the instructions to update MSYS2 after installing it.
+    
+    1. Open the appropriate MINGW shell (e.g., `C:\msys64\mingw32.exe` or `C:\msys64\mingw64.exe`, depending on whether you want to build 32- or 64-bit binaries) and install the compile-time dependencies:
+
+        ```
+        pacman -Sy make ${MINGW_PACKAGE_PREFIX}-gcc ${MINGW_PACKAGE_PREFIX}-cmake ${MINGW_PACKAGE_PREFIX}-mbedtls
+        ```
+
+    **Compile the project**
+
+    Run the following commands in the MINGW shell, after navigating to the directory created using Git above:
+
+    ```
+    cmake -G"MSYS Makefiles" -DDTLS_BACKEND="mbedtls" .
+    make
+    ```
+
+## Add a new device in Coiote IoT DM
+
+After compiling the demo project, go back to the [Coiote IoT DM portal](https://eu.iot.avsystem.cloud/).
+
+Click the button **+ Add device** in the top-right corner.
+
+![Add Device button](img/add-device.png)
+
+Add a new device using the option: **Anjay LwM2M Client Demo**
+
+![Anjay LwM2M Client Demo](img/add-demo-device.png)
+
+Copy the command you find in Coiote.
+
+![Run command in terminal](img/run-command.png)
+
+Go back to your terminal, open your Anjay directory and **run the command** to configure and activate the demo application.
+
+## Monitor your demo device in the Coiote IoT DM
+
+If the demo application started successfully, a device is shown in the Coiote platform.
+
+![Coiote dashboard](img/coiote-dashboard.png)
+
+### Review the Data model
+
+Click the tab **Data model** from the top-menu.
```

Comment:
`Go to the **Data model** tab in the top-menu.` - it's better to avoid `click` in such contexts